src/serverUtils.js

```diff
@@ -41,7 +45,7 @@ const BaseServer = class {
     
     client.connect(port, host, ()=>{
       console.log(`PEER connecting: ${remoteAddr}`);
-      client.write(`connect**@#@${port}`);
+      client.write(`connect**@#@${this.serverPeerPort}`);
```

Comment:
Nice catch. Is this a protocol change, to write back to the client the serverPeerPort instead of the client's port?

Reply:
Actually, if we don't send the peer the port this server is listening for peer connection, other peers who receive the peer list from this peer will not be able to connect with this server. Hope that makes sense.

Reply:
oh nice. we just have never had more than two peers to discover this phenomenon. makes sense